act-rules-format.bs

```diff
@@ -83,7 +83,7 @@ Rule Identifier {#rule-identifier}
 An ACT Rule MUST have a unique identifier that can be any unique text, such as plain text, URL or a database identifier.
 
 <div class=example>
-  <p>ACT Rules in the Auto-WCAG community groups use file names as identifiers. They include a number for a WCAG 2.1 success criterion, and a handle that includes an element name or attribute:</p>
+  <p><strong>Example</strong>: ACT Rules in the Auto-WCAG community groups use file names as identifiers. They include a number for a WCAG 2.1 success criterion, and a handle that includes an element name or attribute:</p>
```

Comment:
Suggested edit: 'ACT Rules developed by the Auto-WCAG community groups use..."  

Also, to clarify understanding of the 2nd sentence...Do you mean, "They include the number corresponding to a WCAG 2.1 success criterion, combined with a handle that includes an element name or name being tested:"
Does that change characterize it more clearly without changing your meaning?